### ISSUE TITLE
Feature Request: Can this software provide the version of Win32?

### ISSUE THREAD

**Issue by ShangJixin** [ShankarBUS/ModernFlyouts#126]
**Is your feature request related to a problem? Please describe:**

It's too easy for UWP applications to crash. I think that the Win32 Application can improve this situation.
UWP应用太容易崩溃了，我认为Win32应用可以改善这种情况。

**Describe the solution you'd like:**

**Describe alternatives you've considered:**

**Additional context:**

**Comment by ShankarBUS** [comment]
Unfortunately we can't do that 😅.

We tied up our app to a packaged environment. Some of the current features (Run at Startup, App data) are hard coded to work with the packaged environment.

We can't do additional work to make it work unpackaged. It will require extra work to maintain two distribution models. We won't be able to track bugs efficiently.

Our app is not a **UWP** app to be clear. It is a packaged **Win32** app.

What you wanted to say must be, "provide an unpackaged distribution of ModernFlyouts".

The crashes will still happen even if the app is unpackaged. The distribution doesn't have anything to do with the crashes or performance.

We chose MSIX due to its easy deployment process. Publishing to store is easy with MSIX. We have the benefits of auto updates, persistent app data from old version to new version, resetting app data from Windows settings, Controlling the startup task from Windows settings and much more.

TL;DR

We don't have the motive to switch back to unpackaged modal
